pkg/apis/jvmbuildservice/v1alpha1/dependencybuild_types.go

```diff
@@ -142,6 +142,7 @@ type BuildRecipe struct {
 	AdditionalMemory    int                  `json:"additionalMemory,omitempty"`
 	Repositories        []string             `json:"repositories,omitempty"`
 	AllowedDifferences  []string             `json:"allowedDifferences,omitempty"`
+	DisabledPlugins     string               `json:"disabledPlugins,omitempty"`
```

Comment:
This should really be a string array. I know at the moment it is just used as a comma separated list but it is still better to preserve the structure.